packages/@react-native-windows/tester/src/js/utils/RNTesterList.windows.js

```diff
@@ -12,6 +12,10 @@ import type {RNTesterModuleInfo} from '../types/RNTesterTypes';
 import ReactNativeFeatureFlags from 'react-native/Libraries/ReactNative/ReactNativeFeatureFlags';
 
 const Components: Array<RNTesterModuleInfo> = [
+  {
+    key: 'XHRExample',
+    module: require('../examples/XHR/XHRExample'),
+  },
```

Comment:
Does the addition of this array element amount to a test, or is actual test code needed to access/exercise this element?

Reply:
It enables existing test code not used by default:
<img width="752" alt="image" src="https://user-images.githubusercontent.com/4507319/199856826-625ef039-5ef2-408d-8db8-69f718d38a7d.png">